src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/InteropGenerationOptions.cs

```diff
@@ -7,5 +7,5 @@
 
 namespace Microsoft.Interop
 {
-    public readonly record struct InteropGenerationOptions(bool UseMarshalType);
+    public readonly record struct InteropGenerationOptions(bool UseMarshalType, bool RuntimeMarshallingDisabled);
```

Comment:
I think this new information would fit better on the `StubCodeContext`.

Reply:
My thought was that we currently don't need that information beyond creating the generator, so I kept it off the `StubCodeContext` and put it on the options passed to the generator factory - similar to `AttributedMarshallingModelGeneratorFactory`.